src/core/lib/transport/timeout_encoding.cc

```diff
@@ -112,8 +113,9 @@ int grpc_http2_decode_timeout(grpc_slice text, grpc_millis* timeout) {
   }
   if (!have_digit) return 0;
   /* skip whitespace */
-  for (; p != end && *p == ' '; p++)
+  for (; p != end && *p == ' '; p++) {
     ;
```

Comment:
No need for this anymore, now that there are braces around it.

Reply:
Done